ion-schema/src/constraint.rs

```diff
@@ -1140,7 +1140,7 @@ impl ConstraintValidator for ContainerLengthConstraint {
         let size = if let Some(element_iter) = value.as_sequence_iter() {
             element_iter.count()
         } else if let Some(strukt) = value.as_struct() {
-            strukt.fields().map(|(k, v)| v).count()
+            strukt.fields().count()
```

Comment:
🗺️ This is a clippy warning that keeps showing up. Since I'm touching this file, I figured I'd fix it now.

Reply:
Is this different from `strukt.len()`?

Reply:
No. I'll change it.